lib/simulator-xcode-11.4.js

```diff
@@ -104,6 +105,25 @@ class SimulatorXcode11_4 extends SimulatorXcode11 {
   async clearKeychains () {
     await this.simctl.resetKeychain();
   }
+
+  /**
+   * Opens simulator's UI Client if not already open and boots sim
+   *
+   * @param uiClientPid - process id of simulator UI client.
+   * @param opts - arguments to start simulator UI client with.
+   * @returns {Promise} - a promise that resolves when sim UI client is launched and sim is booted.
+   * @override
+   */
+  async launchSimWindow (uiClientPid, opts) {
+    // In xcode 11.4, UI Client must be first launched, otherwise
+    // sim window stays minimized
+    if (!uiClientPid) {
+      await this.startUIClient(opts);
+    }
+    log.info(`Booting Simulator with UDID '${this.udid}'...`);
```

Comment:
this log line could be moved to `boot` method

Reply:
Good catch, moved it.